extensions/notification/queueProcessor/QueueProcessor.js

```diff
@@ -45,6 +47,12 @@ class QueueProcessor extends EventEmitter {
         this.bnConfigManager = null;
         this._consumer = null;
         this._destination = null;
+        /**
+         * Once the notification manager is initialized
+         * this will hold the callback version of the getConfig
+         * function of the notification config manager
+         */
```

Comment:
should not be a doc comment?

```suggestion
         // Once the notification manager is initialized
         // this will hold the callback version of the getConfig
         // function of the notification config manager
```